### DISCUSSION TITLE
Dịch `ensemble (learning)`

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#6] [Terminology] [answered]
Mọi người có gợi ý gì cho cách dịch `ensemble` không?  Một vài gợi ý:
- quần thể
- hợp tác

cc: @ngcthuong @rootonchair @nguyenhoa93 @quangnhat185 @tiepvupsu @minhduc0711 @davidnvq @duythanhvn

**Comment by lkhphuc** [comment, marked as answer]
Mình thấy cụm `ensemble model` hay được xài hơn `ensemble learning` thì phải.
`Mô hình kết hợp` thì sao?

**Reply by quangvnai** [reply]
`Mô hình kết hợp` thấy xuôi tai hơn hai gợi ý trên nhưng không biết từ kết hợp ở đây là kết hợp nhiều models (theo ensemble) hay nhầm sang kết hợp các components bên trong model (ví dụ: multimodal fusion model, fused model). Có khi giữ nguyên cái từ `ensemble`?

**Reply by lkhphuc** [reply]
Vậy chắc tạm thời không dịch `ensemble` nhé.

**Reply by ngcthuong** [reply]
Ensemble dịch là ghép?
